typescript/sdk/src/token/altVMDeploy.ts

```diff
@@ -66,6 +68,10 @@ export class CosmosNativeDeployer {
           result[chain] = await this.deploySyntheticToken(
             chain,
             config.mailbox,
+            config.name,
```

Comment:
More typing errors: `config.name` is optional, and `config.description` doesn't exist on synthetic token types.